.circleci/config.yml

```diff
@@ -30,13 +30,13 @@ jobs:
           mkdir -p reports
           npm run ci-test-lint
 
-    - run:
-        name: coverage
-        environment:
-          COVERALLS_REPO_TOKEN: "9tBxvG8hoJroIbSNdMMNY9kNKTFVgVhQd"
-        command: |
-          npm run cov
-          npm run cov-html
+#    - run:
```

Comment:
Could you add a comment as to why it was disabled? It's hard to understand just from looking at it, what the intention was. Questions: Why was it disabled? When will it be ready to be reenabled?

Reply:
I have added a link to the coverage issue I had https://github.com/gotwarlost/istanbul/issues/602. We have now 2 possibilities ahead of us:

- Wait that the issue will be solved directly in the istanbul core
- Trying to find out why this issue appears in in vue-ui. In this case I have spent already too much time on it and I had to opt in for a pragmatic solution in the short term

Reply:
@GianlucaGuarini 
That's fine. It's just a suggestion for future issues like these :)

I just wanted to point out that it's important that such information is delivered via comment. Else it's forgotten and lost in the flow.
Even with a link to an issue, it's not really clear what this does. Was it commented out to solve the issue? Is it a workaround or a fix? Is it temporary or will ist stay that way for the time being?